scripts/agentic-tdd/wave1-complete.sh

```
@@ -0,0 +1,188 @@
+echo "🔍 Validating task file structure..."
+invalid_tasks=()
+
+for task_file in shared/artifacts/tasks/*.md; do
+    [ "$task_file" = "shared/artifacts/tasks/TEMPLATE-task.md" ] && continue
+    [ ! -f "$task_file" ] && continue
+    
+    # Check for required sections
+    if ! grep -q "## Description" "$task_file" || \
+       ! grep -q "## Acceptance Criteria" "$task_file" || \
+       ! grep -q "## Dependencies" "$task_file"; then
+        invalid_tasks+=($(basename "$task_file"))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)